packages/channel-client/src/channel-client.ts

```diff
@@ -343,6 +344,7 @@ export class ChannelClient implements BrowserChannelClientInterface {
    * @returns A promise that resolves to a DomainBudget.
    *
    */
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
What's this doing?

Reply:
It's a new rule introduced in the upgrade https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md . It objects to our use of `{}` as a type. 

Rather than just globally ignoring this rule, I thought it better to ignore in specific places, giving us a greater chance to remedy in the future.